addons/compat_gm/compat_gm_explosives/config.cpp

```diff
@@ -11,7 +11,12 @@ class CfgPatches {
             "ace_explosives"
         };
         skipWhenMissingDependencies = 1;
+        author = ECSTRING(common,ACETeam);
+        authors[] = {};
+        url = ECSTRING(main,URL);
         VERSION_CONFIG;
+
+        addonRootClass = QUOTE(COMPONENT);
```

Comment:
Regarding this, I am not sure if we want it?

[`CfgPatches` wiki page](https://community.bistudio.com/wiki/CfgPatches) says:
> To make configuration of meta data simpler, you can simply create a link to the root class instead of defining name, author and url again. A good practice is to have all configs withing one PBO to use the same meta data.

But some of our compats might be better off requiring a subcomponent specifically?